src/Queue/SearchNoHitsProcessor.php

```diff
@@ -63,8 +64,8 @@ public function process(Message $message, Context $session)
         // If it's a "katalog" identifier, we will try to check if a matching
         // "basic" identifier exits and create the mapping.
         if (strpos($identifier, '-katalog:')) {
-            $parts = explode(':', $identifier);
-            $basicPid = $this::BASIC_PID_PREFIX.end($parts);
+            $faust = Material::translatePidToFaust($identifier);
+            $basicPid = self::BASIC_PID_PREFIX.$faust;
```

Comment:
Would prefer that `BASIC_PID_PREFIX` is also moved to the `App\Utils\OpenPlatform\Material` utility class, then let that class do `katalog` to `basis` conversion

Reply:
Fixed